commands/commands.go

```diff
@@ -12,6 +10,9 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
+
+	"github.com/github/git-lfs/lfs"
+	"github.com/spf13/cobra"
```

Comment:
If we're going to go with this style of import block (which I am in favor of, and is the only annoying difference between `goimports` and `go fmt`), then we should do it this way everywhere.